### DISCUSSION TITLE
Research on Alternate Education Paradigms

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#15] [Q&A]
What research/evidence exists to support evidence for transformative adult education?

**Comment by avidrucker** [comment]
@seekanddefine In response to `"What research/evidence exists to support evidence for transformative adult education"` I did some online reading into it. If I understand my readings, the evidence is not yet well established. This summary is the main bulk of my reading so far (it has the full research paper linked, and is immediately available for download as a docx file): https://app.oxfordabstracts.com/events/877/program-app/submission/132173

**Reply by avidrucker** [reply]
As a further reading / research endeavor, I believe this may shine some more like on the subject, with a connection / overlap with topics such as "mindfulness" and "acceptance" ... that said, I don't believe there is any empirical evidence sited here: https://files.eric.ed.gov/fulltext/EJ1257177.pdf

**Reply by avidrucker** [reply]
OK, here is some interesting data! https://files.eric.ed.gov/fulltext/EJ1280740.pdf This reminds me of a resource I'd like to share with you that I made.

**Reply by avidrucker** [reply]
I like how this summarizes, make a little bit clearer and accessible the concepts of TL to me, and also touches on collaborative learning (in contrast with the study just above, which focuses on remote solo learning) https://360learning.com/blog/transformative-learning/

**Reply by avidrucker** [reply]
What a rabbit hole 😅 Seriously though, this research is compelling. Transformative learning theory in Korea: https://link.springer.com/article/10.1007/s12564-017-9481-x

**Reply by avidrucker** [reply]
OK... I think I'm good for now on researching TL 😅 https://theo-dawson.medium.com/transformational-learning-revisited-5e3e1ae02c26

**Comment by avidrucker** [comment]
What research exists to support evidence for collaborative learning? ( note for context: I am being introduced to the formal concept here: https://360learning.com/collaborative-learning/ )

**Comment by avidrucker** [comment]
> What research/evidence exists to support evidence for transformative adult education?

@seekanddefine May we change the question prompt to allow for multiple threads of research? For example, to investigate different educational paradigms, techniques, etc.? So, specifically, may we change `What research/evidence exists to support evidence for transformative adult education?` to `Please add an educational technique, concept, or paradigm to research supporting evidence` ?
